src/js/common/schemaform/widgets/TextWidget.jsx

```diff
@@ -12,7 +12,7 @@ export default function TextWidget(props) {
         maxLength={props.schema.maxLength}
         autoComplete={props.options.autocomplete || false}
         className={props.options.widgetClassNames}
-        value={props.value || ''}
```

Comment:
It doesn't look like this change is affecting anything. The empty string should be turned into undefined by the onChange function below.